src/chainstate/stacks/miner.rs

```diff
@@ -445,23 +726,21 @@ impl<'a> StacksMicroblockBuilder<'a> {
         self.runtime.num_mined = num_txs;
 
         match result {
-            Err(Error::BlockTooBigError) => {
```

Comment:
I moved this code block to the `ProcessingError` branch of the match just above. 
This piece of code was actually never executed before, because `mine_next_transaction` never returned an Err.